code/modules/surgery/organs/internal/lungs/_lungs.dm

```diff
@@ -780,20 +786,39 @@
 
 /obj/item/organ/internal/lungs/on_life(seconds_per_tick, times_fired)
 	. = ..()
-	if(failed && !(organ_flags & ORGAN_FAILING))
+	if(failed && can_breathe(only_check_failure_states = TRUE))
 		failed = FALSE
 		return
 	if(damage >= low_threshold)
 		var/do_i_cough = SPT_PROB((damage < high_threshold) ? 2.5 : 5, seconds_per_tick) // between : past high
 		if(do_i_cough)
 			owner.emote("cough")
-	if(organ_flags & ORGAN_FAILING && owner.stat == CONSCIOUS)
+	if(!failed && owner.stat == CONSCIOUS && !can_breathe(only_check_failure_states = TRUE))
 		owner.visible_message(span_danger("[owner] grabs [owner.p_their()] throat, struggling for breath!"), span_userdanger("You suddenly feel like you can't breathe!"))
 		failed = TRUE
 
 /obj/item/organ/internal/lungs/get_availability(datum/species/owner_species, mob/living/owner_mob)
 	return owner_species.mutantlungs
 
+/obj/item/organ/internal/lungs/proc/can_breathe(only_check_failure_states = FALSE)
+	if (organ_flags & ORGAN_FAILING)
+		return FALSE
+	if (received_pressure_mult <= 0) // i mean, its literally impossible to get you to breathe when no air comes in
+		return FALSE
+	if (!only_check_failure_states)
+		if (HAS_TRAIT(owner, TRAIT_NOBREATH))
+			return FALSE
+	return TRUE
+
+/obj/item/organ/internal/lungs/proc/currently_breathing()
```

Comment:
Docs, also see before about clarifying what breathing means

Reply:
ill be honest, im mostly struggling to figure out whats actually breathing and what isnt, so i just said fuck it and added a trait that signifies what breathes by default, should root out most edge cases of things